tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs

```diff
@@ -96,6 +97,19 @@ public ActionCenterMessages GetRebootMessage()
         };
     }
 
+    /// <summary>
+    /// Get the arguments for this task
+    /// </summary>
+    /// <returns>Arguments for this task</returns>

```

Comment:
Nit: expand comment to clarify it's the arguments that are passed to the elevated process